frontend/src/models/frontend_models.ts

```diff
@@ -106,7 +106,9 @@ export function mergeItemsByID<T extends FMSTrackedModel>(itemsByID: ItemsByID<T
 }
 
 export type ObjectId = FMSId
-export interface Container extends Readonly<FMSContainer>, FetchedObject {}
+export interface Container extends Readonly<FMSContainer>, FetchedObject {
+	parents?: Container["id"][]
```

Comment:
Does not appear right to define it here ... wouldn't this be better in fms_api_model FMSContainer ? this seem like a code bit that is there to feed the lint-monster ...

Reply:
They way I look at it, `fms_api_model` is strictly for defining fields that are present the response from the API. The `parents` field is actually created in the frontend.

Reply:
Yeah as long as we use that representation everywhere in the frontend that matters ... Not sure it was done that way elsewhere (seeing as you created the only deviation from the backend object).